sig-scalability/README.md

```diff
@@ -78,10 +78,15 @@ Note that the links to display team membership will only work if you are a membe
 
 <!-- BEGIN CUSTOM CONTENT -->
 # SIG-Scalability Team Members
-* Janek Łukaszewicz (**[@oxddr](https://github.com/oxddr)**), Google
+
```

Comment:
Maybe we should change the title?
To sth like scalability-regression contact-points? or sth like that?

Reply:
Done.

Reply:
Not really yet :)

Reply:
Doh, done :)